### ISSUE TITLE
Same erasure when expanding @Odd annotation

### ISSUE THREAD

**Issue by jpotecki** [jlink/jqwik#39]
## Testing Problem

I followed the documentation for the implementation for `@Odd` but quickly ran into an issue when I wanted to add a `configure` method for `BigInteger`.
```
configure(Arbitrary<BigInteger>,Odd) and configure(Arbitrary<Integer>,Odd) have the same erasure
```

## Suggested Solution

It would be great the documentation could give some more instructions how to create such an overloaded `configure` method.

**Comment by jpotecki** [comment]
good idea for the workaround! Will do that for now :)

**Comment by jlink** [comment]
Commit https://github.com/jlink/jqwik/commit/06512504871b8cd4bc4904703b297fb80f839129 adds a possibility to accept only certain target types.

See reworked user-guide: https://jqwik.net/user-guide.html#arbitrary-configuration-example-odd

Will be available in 0.9.3. Already available in 0.9.3-SNAPSHOT.

**Comment by jlink** [comment]
Feel free to reopen the issue if solution does not fit your problem.
